docs/content/en/options.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Options
+description: a list of configuration options for Google Analytics in Nuxt
+position: 3
+category: Guide
+fullscreen: false
+---
+
+To configure the module, you can use `googleAnalytics` section in `nuxt.config.js`.
+
+```js[nuxt.config.js]
+export default {
+  googleAnalytics: {
+    // Options
+  }
+}
+```
+
+<alert type="info">
+
+`router` instance is added out of the box. You can refer [here](https://github.com/MatteoGabriele/vue-analytics/blob/master/docs/page-tracking.md#disable-page-auto-tracking) on to how to disable it if needed.
+
+</alert>
+
+## Google Analytics options
+
+### `id`
+
+* Type: `String`
+* **Required**
+
+The tracking ID of your Google Analytics account. It is required to have Google Analytics (GA) know which account and property to send the data to.
+
+```js[nuxt.config.js]
+export default {
+  googleAnalytics: {
+    id: 'UA-XXX-X'
+  }
+}
+```
+
+<alert type="info">
+
+For backwards compatibilities, use `ua` instead.
+
+</alert>
+
```

Comment:
Can add asyncID option?
https://github.com/nuxt-community/analytics-module/issues/85

Allow an asynchronous function to load the id
https://github.com/nuxt-community/analytics-module/blob/master/lib/plugin.js#L9